lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -105,12 +105,12 @@ class SPIRVDecorateGeneric:public SPIRVAnnotationGeneric{
   SPIRVDecorationGroup *Owner; // Owning decorate group
 };
 
-class SPIRVDecorateSet: public std::multiset<const SPIRVDecorateGeneric *,

```

Comment:
Why did you remove "const" qualifier?

Reply:
This is basically because in SPIRVModule.h addEntry function does not take const SPIRVEntries. 
If you take a look at other similar classes, e.g. SPIRVDecorationGroup, they also do not have const qualifiers along the way.